Documentation/EyeTracking/EyeTracking_BasicSetup.md

```diff
@@ -92,32 +100,7 @@ Now that your scene is set up to use eye tracking, let's take a look at how to a
 
 ### Testing your Unity app on a HoloLens 2
 
-Building your app with eye tracking should be similar to how you would compile other HoloLens 2 MRTK apps.
-The only difference is that the *'Gaze Input'* capability is unfortunately not yet supported by Unity under 'Player Settings -> Publishing Settings -> Capabilities'.
-To use eye tracking on your HoloLens 2 device, you need to manually edit the package manifest that is part of your built Visual Studio project.
-
-Follow these steps:
-
-1. Build your Unity project as you would normally do for _HoloLens 2_.
-2. Open your compiled Visual Studio project and then open the _'Package.appxmanifest'_ in your solution.
-3. Make sure to tick the _'GazeInput'_ checkbox under _Capabilities_.
-
-_Please note:_
-You only have to do this if you build into a new build folder.
-This means that if you had already built your Unity project and set up the appxmanifest before and now target the same folder again, the appxmanifest should stay untouched.
-
-![Enabling Gaze Input in Visual Studio](../Images/EyeTracking/mrtk_et_gazeinput.jpg)
-
-You don't see a _'GazeInput'_ capability?
-
-- Check that your system meets the [prerequisites for using MRTK](../GettingStartedWithTheMRTK.md#prerequisites) (in particular the Windows SDK version).
-- You can also manually add the entry by opening the appxmanifest in an XML editor and adding the following:
-
-```xml
-  <Capabilities>
-    <DeviceCapability Name="gazeInput" />
-  </Capabilities>
-```
+Building your app with eye tracking should be similar to how you would compile other HoloLens 2 MRTK apps. Be sure that you have enabled the *'Gaze Input'* capability as described above in the section *A note on the GazeInput capability*.
```

Comment:
Might be useful to add a link (I think this is the correct syntax)

```suggestion
Building your app with eye tracking should be similar to how you would compile other HoloLens 2 MRTK apps. Be sure that you have enabled the *'Gaze Input'* capability as described above in the section [*A note on the GazeInput capability*](#a-note-on-the-gazeinput-capability).
```

Reply:
Thanks @keveleigh , I didn't know how to do a section link and now I do. I verified that your syntax works on [the page](https://github.com/microsoft/MixedRealityToolkit-Unity/blob/975cca3adc086f24c03f21120588a322897c34a0/Documentation/EyeTracking/EyeTracking_BasicSetup.md#testing-your-unity-app-on-a-hololens-2).

Reply:
Awesome, thanks!!